live-site/components/event-schedule-section/event-schedule-tab/EventScheduleTab.tsx

```diff
@@ -12,11 +15,59 @@ import {
   StyledEventScheduleTabContainer
 } from './EventScheduleTab.styles';
 
+const AIRTABLE_BASE_NAME = 'Schedule';
+const AIRTABLE_SCHEDULE_TABLE_NAME = 'schedule';
+const FRI_DATE = new Date(2023, 1, 10);
+const SAT_DATE = new Date(2023, 1, 11);
+const SUN_DATE = new Date(2023, 1, 12);
+
+const renderEventData = (events: any[], activeTab: number): JSX.Element[] => {
+  const toEventItemType = (event: any): EventItemType => {
+    return { id: event.id, ...event.fields };
+  };
+  const convertStringToDate = (event: EventItemType): EventItemType => {
+    console.log(event.time);
+    return { ...event, time: new Date(Date.parse(event.time as string)) };
+  };
+  const filterByDate = (event: EventItemType): boolean => {
+    switch (activeTab) {
+      case 0:
+        return event.time < SAT_DATE && event.time > FRI_DATE;
+      case 1:
+        return event.time < SUN_DATE && event.time > SAT_DATE;
+      case 2:
+        return event.time > SUN_DATE;
+      default:
+        throw Error('Invalid tab.');
```

Comment:
should we be throwing an error here? might be more graceful to do nothing

Reply:
I disagree, silent errors will be hard to debug. An error will allow peeps to hotfix. However, so now whenever changes the constraint that the tabs aren't 0,1,2 they'll know that they broke the scheduler in a pretty noticeable way.

Reply:
bet, just unsure since we don't throw an error anywhere else in the codebase